docker/.env.example

```diff
@@ -1,18 +1,28 @@
 # Secrets
 # YOU MUST CHANGE THESE BEFORE GOING INTO PRODUCTION
+# used as a secret to verify the JWT token signature
 JWT_SECRET=your-secret
-STORE_ENCRYPTION_KEY="<ENCRYPTION_KEY_MUST_BE_32_LONG>"
+# used to encrypt/decrypt the provider credentials
+STORE_ENCRYPTION_KEY=<ENCRYPTION_KEY_MUST_BE_32_LONG>
+
+# Host
+HOST_NAME=http://localhost
 
 # General
-NODE_ENV=local
-MONGO_URL=mongodb://mongodb:27017/novu-db
+NODE_ENV=production
```

Comment:
i think the default should be `local`, and will be updated to production only if it was intentional in order to reduce human mistakes. wdyt?

Reply:
Do you have any ref of this error? or why it gives 500, it should not happen with local NODE_ENV